README.md

```diff
@@ -178,6 +179,16 @@ Represents a built Docker image
 
 ```python
 class Image
+  def cache(path: string):
```

Comment:
I've been trying to specify when functions don't return anything. If we don't want to do that we should remove the other references to `None` in the README.

```suggestion
  def cache(path: string)  -> None:
```

Reply:
done!